### DISCUSSION TITLE
persistent variables - proposed for northstar 1.5

### DISCUSSION THREAD

**Discussion by xia0** [ScureX/Takyon.PlayerVote#23] [General]
i see that it is a planned feature. ive been advised in discord that this is planned for northstar 1.5 release.  might be better to wait for more streamlined implementation.

**Comment by ScureX** [comment]
1.5 just released this minute! AFAIK this feature cannot yet be implemented as there is no support for writing/reading files outside of squirrel scripts (might come in 1.5.1 in like a week tho). The main point of this feature is stuff like making temp bans (currently reset after map change) but more importantly making a config so settings can be applied while the server is still up (adding new admins, etc). If you have a good idea for this feature pls lmk!
